plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -864,8 +864,14 @@ function VocabItemWidget:getTimeSinceDue()
         readable_time = string.format("%d"..C_("Time", "m"), rounding(abs/60))
     elseif abs < 3600 * 24 then
         readable_time = string.format("%d"..C_("Time", "h"), rounding(abs/3600))
-    else
+    elseif abs < 3600 * 24 * 30 then
         readable_time = string.format("%d"..C_("Time", "d"), rounding(abs/3600/24))
+    elseif abs < 3600 * 24 * 365 then
+        local mo = rounding(abs/3600/24/3)/10
+        readable_time = string.format((mo==rounding(mo) and "%d" or "%.1f")..C_("Time", "mo"), mo)
```

Comment:
`mo` and `yr` are fine, but I just noticed I let a string.format slip through instead of http://koreader.rocks/doc/modules/ffi.util.html#template

```
plugins/vocabbuilder.koplugin/main.lua
864:        readable_time = string.format("%d"..C_("Time", "m"), rounding(abs/60))
866:        readable_time = string.format("%d"..C_("Time", "h"), rounding(abs/3600))
868:        readable_time = string.format("%d"..C_("Time", "d"), rounding(abs/3600/24))
```

Reply:
Should I change them into templates?

Reply:
Done